levels/utils.go

```diff
@@ -24,9 +44,37 @@ func print_line(text string) {
 }
 
 func PrintText(text string) {
+	var echo_state bool = true
+	sigs := make(chan os.Signal, 1)
+	signal.Notify(sigs, syscall.SIGINT, syscall.SIGTERM)
+	go func() {
+		<-sigs
+		if echo_state == false {
+			exec.Command("stty", "-F", "/dev/tty", "echo").Run()
+		}
+		os.Exit(0)
+	}()
+
+	// disable input buffering
+	err := exec.Command("stty", "-F", "/dev/tty", "cbreak", "min", "1").Run()
+	if err == nil {
```

Comment:
@Foxiy This does not seem to change much in my view but let's leave it this way up until someone will have the time/energy to fully assess this.